include/net/netutils.h

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
We have already net_ip.h which contains various helper functions, so there is no need to create a new header file for just these.

Reply:
> We have already net_ip.h which contains various helper functions, so there is no need to create a new header file for just these.

So helper functions are for Zephyr network stack. These are application-level helpers on top of POSIX API.

Reply:
Why not call this `utils.h` as the net is redundant info here as the file is in net directory anyway. We have other net_ prefixed header files in the net directory but it feels unnecessary now, unfortunately we cannot really change that at this point.

Reply:
> Why not call this utils.h as the net is redundant info here as the file is in net directory anyway.

As I mentioned, the idea was to make it have more selective name than just often-overused "utils", in particular, because of the idea to use it e.g. in POSIX (Linux) app builds, where "net/" prefix might be there or not.

Calling it just net/utils.h might work, but based on the previous discussion, it seems it's worth emphasizing that it's socket API level utils, so let me try socketutils.h and see if that works. (Also considered sockutils.h, but that may be confused with utils for SOCKS protocol for example).